tests/format/typescript/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -543,8 +543,8 @@ abstract class Test {
 =====================================output=====================================
 interface foo1 {
   bar3 /* foo */(/* baz */); // bat
-  bar /* foo */ /* bar */?(/* baz */) /* bat */;
-  bar2 /* foo */(/* baz */) /* bat */;
+  bar /* foo */ /* bar */?(/* baz */); /* bat */
+  bar2 /* foo */(/* baz */); /* bat */
```

Comment:
Is this change in the output intended?

Reply:
I think so, 

**Prettier 2.8.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBLWcBOAzAhmOAAmwggEZDgAdKQwgI100IHoAqB3GQtlgbhp1GzPu07deNAL40aYADa4AzkuKkK1Wp2ZjGE-oO2FRHPTxbTZUEABoQEAA4xU0JclBNMEAO4AFJghuKLjy3rgAnm529Jj4ANZwMADKuAC2cAAy6HDIePJKcNGxYAnJDvjoAObIMJgAroUgcKn0cAAmbe0ZuFCVdbiVcABiEJipXM69yCC4dTAQtiAAFjCp8gDqS6jwSuUESYHbqABu2+HTYCqL6AWYML6xleO5IQV2AFZKAB5JVfJwAEU6hB4C98o1yphbtMYOEHHAlGBMKgnIsHMjYOtUG0YEtkAAOAAMdnREAK61iDmm6IRWGOOTsAEdgfAHo4gjMlABaKBwdrtRaYODM1BCh4DZ5IPJvEAFVKoGr1RpKP5wACCMFqqHoczgviwWV5YJlKt6-yBIJyUtejRguHoWJxeKQACY7LVcKh5FUAMIQVKSppKACsizqBQAKvagtLGscGgBJKCdWBJJEomBq5NJWH-Y1wKRSIA)
<!-- prettier-ignore -->
```sh
--parser typescript
```

**Input:**
<!-- prettier-ignore -->
```tsx
interface foo1 {
  bar /* bat */;
  bar ;/* bat */
}

class foo1 {
  bar /* bat */;
  bar ;/* bat */
}


```

**Output:**
<!-- prettier-ignore -->
```tsx
interface foo1 {
  bar /* bat */;
  bar /* bat */;
}

class foo1 {
  bar /* bat */;
  bar; /* bat */
}

```